browser/scripts/util.js

```diff
@@ -198,3 +198,6 @@ E2.util = {
 		return !!(document.mozFullScreenElement || document.webkitFullscreenElement);
 	}
 }
+
+if (typeof(module) !== 'undefined')
+	module.exports = load_script
```

Comment:
I'd do a `exports.load_script = load_script` so we can add more exports to this module